Logger/Person.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Logger;
+
+public record class Person : BaseEntity
+{
+    //Implemented Name implcitily since FullName is never saved as a property itself so it cant be accessed, so no conflicts
+    public override string Name { get; set; }
+
+    public Person(FullName name)
+    {
+        Name = name.ToString();
+    }
```

Comment:
This doesn't check if FullName is null.
```suggestion
    public Person(FullName name)
    {
ArgumentException.ThrowIfNullOrWhiteSpace(name);        Name = name.ToString();
    }
```
This is if you decide to stay with this constructor. If you do a primary constructor, you can do a null coalescing operator for null-checking for name

Reply:
nice catch will fix